packages/block-editor/src/components/page-template-picker/picker.js

```diff
@@ -25,6 +30,9 @@ const __experimentalPageTemplatePicker = ( {
 			blocks: parse( templatePreview.content ),
 		} );
 		setTemplatePreview( undefined );
+		logUserEvent( userEvents.editorSessionTemplateApply, {
+			template: templatePreview.name,
```

Comment:
I don't know why but I thought we had a template "slug" already. The template name can be localized, so it'd be good to add some non-translated string that uniquely identifies each template and send that instead.

Reply:
I think the `templatePreview` here might be the slug you're thinking of. It's just labeled differently. https://github.com/WordPress/gutenberg/pull/20317/files/1adc58f3836e3a414c231aba42eb4b2fa51d3606#diff-5ac945ef99119bbaeffb91e006ced87bR53 

I could add a `key` field to the default templates and we could use that for non translated logging.

Reply:
That's adjusted now